CONTRIBUTING.md

```
@@ -2,26 +2,31 @@
-* Create a topic branch to work on the issue. Pick a meaningful branch name, e.g. something like `issue/283-rate-limit-login-attempts`.
+* Create a topic branch on top of the integration branch `dev` to work on the issue. Pick a meaningful branch name, e.g. something like `issue/283-rate-limit-login-attempts`. Thee the [diagram of the branching strategy](https://github.com/ude-soco/CourseMapper-webserver/discussions/1289) in use.
 * Work on the branch until the issue is resolved. The usual rules apply: Create atomic commits with meaningful messages, do not include unrelated changes or built artifacts etc.
-* Clean up your branch: Remove unnecessary changes, merge later fixes to previous commits, rebase on top of `main`.
-* Once you're done, create a Pull Request and ask one of the maintainers for a review. The title of the PR should summarize the changes and is usually close to the title of the original issue. The description is mainly intended for the reviewer and should help them understand your approach, possible side effects, dependencies on other features etc.
+* Clean up your branch: Remove unnecessary changes, merge later fixes to previous commits, rebase on top of `dev`.
+* Once you're done, create a Pull Request into `dev` and ask one of the maintainers for a review. The title of the PR should summarize the changes and is usually close to the title of the original issue. The description is mainly intended for the reviewer and should help them understand your approach, possible side effects, dependencies on other features etc.
+* The project maintainers are responsible for merging the integration branch into `main`.
+
+*Note:* Other rules may apply if you're working on the project as part of a student project/thesis. Ask your instructor/supervisor.
+
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)